djcroco/tests/tests.py

```diff
@@ -78,7 +59,7 @@ def test_document_name(self):
     def test_document_size(self):
         # Ensure correct size
         self.assertEqual(self.instance.document.size, 679)
-        self.assertEqual(self.instance.document.size_human, '679 bytes')
+        self.assertEqual(self.instance.document.size_human, u'679\xa0bytes')
```

Comment:
Ohh, ok!  I'll try it.  Do you remember what the syntax for that is?  I can't find it easily.